Comp2023/src/main/java/frc/robot/Robot.java

```diff
@@ -41,6 +42,18 @@ public void robotInit( )
     DataLogManager.start( );
     DataLogManager.log("RobotInit: RoboRIO SN:" + System.getenv("serialnum"));
 
+    if (System.getenv("serialnum").equals("03238074"))

```

Comment:
Can we put these serial number constants into the Constants file? At the very top of the class variables, since it is one of the most fundamental constants.